diff --git a/node_modules/graphiql-explorer/dist/Explorer.js b/node_modules/graphiql-explorer/dist/Explorer.js
index 3bef05d..33c57df 100644
--- a/node_modules/graphiql-explorer/dist/Explorer.js
+++ b/node_modules/graphiql-explorer/dist/Explorer.js
@@ -1580,7 +1580,7 @@ var FieldView = function (_React$PureComponent7) {
             {
               style: { color: styleConfig.colors.property },
               className: 'graphiql-explorer-field-view' },
-            field.name
+            field.name + (String(field.description)==="null" ? "" :' ' + field.description)
           ),
           !this.state.displayFieldActions ? null : React.createElement(
             'button',
diff --git a/node_modules/graphiql-explorer/graphiqlExplorer.js b/node_modules/graphiql-explorer/graphiqlExplorer.js
index e233913..5f3a836 100644
--- a/node_modules/graphiql-explorer/graphiqlExplorer.js
+++ b/node_modules/graphiql-explorer/graphiqlExplorer.js
@@ -1582,7 +1582,7 @@ var FieldView = function (_React$PureComponent7) {
             {
               style: { color: styleConfig.colors.property },
               className: 'graphiql-explorer-field-view' },
-            field.name
+            field.name + (String(field.description)==="null" ? "" :' ' + field.description)
           ),
           !this.state.displayFieldActions ? null : React.createElement(
             'button',
